app/src/main/java/com/android/voyageur/model/trip/TripsViewModel.kt

```diff
@@ -151,23 +213,84 @@ open class TripsViewModel(
         })
   }
 
+  /**
+   * Gets the count of trip notifications.
+   *
+   * @param onSuccess Callback to be invoked on success.
+   */
+  fun getNotificationsCount(onSuccess: (Long) -> Unit) {
+    val userId = Firebase.auth.uid.orEmpty()
```

Comment:
As I see you're calling `Firebase.auth.uid.orEmpty()` to get the `userId` in more than one method, couldn't it be a single `val`?

Reply:
You are right! I've just implemented this change. Thank you for pointing this out, great catch!

Reply:
Alright, I see your point, it's not essential for this PR so I agree with leaving it like this.